src/documentation/io/built-in.md

```diff
@@ -56,6 +60,7 @@ Consult the [Programming Guide I/O section]({{site.baseurl }}/documentation/prog
 <tr>
   <td>Python</td>
   <td>
+    <p>Beam Python supports GCS and local filesystems out of the box. Support for HDFS is in progress (<a href="https://issues.apache.org/jira/browse/BEAM-3099">BEAM-3099</a>).</p>
```

Comment:
Similarly, though not sure about the equivalent dependency statement for Python?
Google Cloud Storage and local filesystems

Reply:
Same.